### DISCUSSION TITLE
Welcome to my Discussions!

### DISCUSSION THREAD

**Discussion by TechStudent10** [TechStudent10/techstudent10#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hungl6844** [comment]
Hi! Your GitHub page seems pretty polished (better than mine, at any rate)! I have a few questions. Uh, heroku. What... is it? It hosts your website, did you write that yourself? (HTML, CSS, etc.) Whatever the case may be, it's cool that you have your own website! (I have a GitHub Pages, but it's very much a WIP...)

**Reply by TechStudent10** [reply]
Hi @SorrowEater! First, [Heroku](https://heroku.com) is a self-hosting platform that allows you to host websites for free! And no I didn't write Heroku. I am not **that** much of an advanced programmer haha. I **did**, however, make the website myself. Heroku doesn't host my Github Pages site. My **actual** Github Pages site just says "I am TechStudent11!" with the Arial font. Here is the link to it: https://techstudent11.github.io.

**Reply by KendallDoesCoding** [reply]
Oh wow, I researched heroku and it seems it was founded in 2007, means its been here from years.
